src/loader/m3u8-parser.ts

```
@@ -367,8 +367,10 @@ export default class M3U8Parser {
             const decryptkeyformat =
               keyAttrs.enumeratedString('KEYFORMAT') ?? 'identity';
 
+            // TBD we might need to check if we try to MSE playback mp2t content
```

Do we need to throw an handled error if someone tries to read MP2T Fairplay-encrypted content ? Or do we only rely on documentation ? 

If there's a need for an error to be thrown, I might need some guidance to see where I should do it :)